examples/Post.php

```diff
@@ -0,0 +1,143 @@
+<?php
+/**
+ * This file is part of the php-state project.
+ *
+ * (c) Yannick Voyer <star.yvoyer@gmail.com> (http://github.com/yvoyer)
+ */
+
+namespace Star\Component\State\Example;
+
+use Star\Component\State\State;
+use Star\Component\State\StateContext;
+use Star\Component\State\StateMachine;
+
+class Post implements StateContext
```

Comment:
Add description in header about how the post is implemented. Add more class example to include event based, closure transition, workflow that communicate state change to the outside world.